math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1058,27 +1058,4 @@ TVEC_EXTERN_VDT_UNARY_FUNCTION(double, fast_atan)
 
 } // End of ROOT NS
 
-namespace cling {
```

Comment:
I believe this was ripped out in the meantime by the RVec folks, addressing `RVec<bool>`...

Reply:
Do you mean we should keep this?